src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishReleaseDataFunction.cs

```diff
@@ -25,16 +44,58 @@ public PublishReleaseDataFunction(IReleaseStatusService releaseStatusService)
             ILogger logger)
         {
             logger.LogInformation($"{executionContext.FunctionName} triggered: {message}");
-            await UpdateStage(message, Started);
-            // TODO EES-866 Run the importer or copy the data from the statistics database
-            // TODO EES-866 to the publicly available statistics database
-            await UpdateStage(message, Complete);
+            var response = await TriggerDataFactoryReleasePipeline(executionContext, message);
```

Comment:
I will add in configuration to disable this function by default so we don't trigger this except for when we want to test it while running locally, and allow this to be enabled in the Development environment for now, also preventing it from running in Test and Production until we are ready.